tests/test_models.py

```diff
@@ -1,90 +1,114 @@
 # -*- coding: utf-8 -*-
-from cms.api import add_plugin, create_page
-from cms.test_utils.testcases import CMSTestCase
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.test import TestCase
 
-from djangocms_style.models import Style, TAG_CHOICES, CLASS_CHOICES
+from djangocms_style.models import (
+    CLASS_CHOICES, TAG_CHOICES, Style, get_templates,
+)
 
 
-class StyleTestCase(CMSTestCase):
+class StyleModelTestCase(TestCase):
 
     def setUp(self):
-        self.superuser = self.get_superuser()
-        self.home = create_page(
-            title="home",
-            template="page.html",
-            language="en",
-        )
-        self.home.publish("en")
-        self.page = create_page(
-            title="help",
-            template="page.html",
-            language="en",
-        )
+        pass
 
     def tearDown(self):
-        self.page.delete()
-        self.home.delete()
+        pass
```

Comment:
Since `setUp` and `tearDown` don't do anything, they don't need to be defined at all.